Tunnel.js

```diff
@@ -276,22 +312,7 @@ Tunnel.prototype = util.mixin(Object.create(_super), /** @lends module:digdug/Tu
 			}
 
 			var request = sendRequest(self.url, { proxy: self.proxy });
-			request.then(
-				function (response) {
-					var decompressor = new Decompress();
-					decompressor.src(response.data)
-						.use(Decompress.zip())
-						.use(Decompress.targz())
-						.dest(self.directory)
-						.run(function (error) {
-							if (error) {
-								reject(error);
-							}
-							else {
-								resolve();
-							}
-						});
-				},
+			request.then((self.dontExtract ? writeFile : decompressFile),
```

Comment:
I like `_postOptions()` a lot and `Tunnel#download(options)` makes a lot of sense. However `SeleniumTunnel#download(options)` is kinda weird because download is responsible for downloading up to 4 files. For example, what would the user expect if they passed `options.url` to `Tunnel` and it works but `SeleniumTunnel` behaves differently. It doesn't make for a uniform API across all of the children.

Because `Tunnel#download(options)` makes sense, but it doesn't make sense on any of the children, I think we should move most of the download logic to an internal method `Tunnel#_downloadFile(options)` and SeleniumTunnel can use that method to download multiple files.

I took a look at the other tunnels and the addition of `_postDownload()` will eliminate their `download()` overrides.